courses/free-courses-hi.md

```diff
@@ -71,7 +73,7 @@
 * [Solidity ^0.8 | Blockchain | In Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiG6_thChXWzolj9BEG-Y0gh) - Pepcoding (YouTube)
 
 
-### <a id="c"></a>C
+### C
```

Comment:
Why this change?

Reply:
in one of previous commit i applied anchor, and i was told to remove it so i thought repo don't need anchors. So, in order to improve the repo i did this.

Reply:
FYI, The anchors will be added on demand to disambiguate semantically